### PR TITLE
Fix issue 156

### PR DESCRIPTION
**Pull request by LakshmiMekala**
This PR addresses dependency chart updates for mysql and aws-efs

### CODE REVIEW

File: cloud/kubernetes/helm/Chart.yaml [hunk 1]
```diff
@@ -16,26 +16,23 @@ version: 1.0.0
...
+    repository: https://kubernetes-sigs.github.io/aws-efs-csi-driver/
+    condition: efs-driver.enabled
   - name: influxdb
     version: 4.8.4
     repository: https://helm.influxdata.com/   
     alias: influxdb
     condition: influxdb.enabled
   - name: grafana
-    version: 6.1.14
+    version: 5.6.7
```
Comment (rameshpolishetti) [inline cloud/kubernetes/helm/Chart.yaml:35]: Any reason to downgrade grafana chart from 6.1.14 to 5.6.7

File: cloud/kubernetes/helm/values.yaml [hunk 2]
```diff
@@ -9,7 +9,7 @@
 # Declare variables to be passed into your templates.
 
 #Required parameter to be passed for all cloud providers
-cpType: minikube      #cpType can be minikube, aws, awsfargate, azure, openshift OR gcp
+cpType: minikube      #cpType can be minikube, aws, awsfargate, awsefs, azure, openshift OR gcp
```
Comment (rameshpolishetti) [inline cloud/kubernetes/helm/values.yaml:12]: I see a new cloud provider type (cpType) "awsefs" added here, do you mean - 
Amazon Elastic File System?
What is the significance of it?
What is the difference between aws & aswefs?

**Review by rameshpolishetti — COMMENTED** [review]
Please note my inline review comments.

File: cloud/kubernetes/helm/Chart.yaml (hunk 1 above)
Comment (LakshmiMekala) [reply cloud/kubernetes/helm/Chart.yaml]: Reverted to older version

File: cloud/kubernetes/helm/values.yaml (hunk 2 above)
Comment (LakshmiMekala) [reply cloud/kubernetes/helm/values.yaml:12]: This flag is added when user is trying to deploy BE app on aws with efs storageclass, as it requires efs csi driver chart to be installed in cluster

**Review by rameshpolishetti — APPROVED** [review]
looks good

**Review by vpatil-tibco — APPROVED** [review]
LGTM

**Comment by rameshpolishetti** [comment]
@vilasshelar please help to validate the PR.

**Comment by tsaxena18** [comment]
Validated updated mysql and aws-efs charts with BE 6.1.0 on Azure and AWS platforms and it went fine. 
Attached log for the reference.
[1.zip](https://github.com/TIBCOSoftware/be-tools/files/6095750/1.zip)
[2.zip](https://github.com/TIBCOSoftware/be-tools/files/6095752/2.zip)

**Pull request merged**